src/test/shell/bazel/bazel_cc_code_coverage_test.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash -eu
+#
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Unit tests for tools/test/collect_cc_code_coverage.sh
+
+
+# Load the test setup defined in the parent directory
+CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "${CURRENT_DIR}/../integration_test_setup.sh" \
+  || { echo "integration_test_setup.sh not found!" >&2; exit 1; }
+
```

Comment:
If you can please set strict mode. Julio wrote a very cool blog post about this: http://julio.meroh.net/2018/03/shell-readability-strict-mode.html

Reply:
It's on the first line of the script
`#!/bin/bash -eu`